donate/core/templatetags/analytics_tags.py

```diff
@@ -4,8 +4,10 @@
 
 
 @register.inclusion_tag('fragments/ga_events.html', takes_context=True)
+# This tag is used to render GA or datalayer event data on the frontend,
+# so it can be picked up by JS.
 def render_ga_event_data(context):
     return {
         'events': context['request'].session.pop('ga_events', []),
-        'data_layer': context['request'].session.pop('data_layer', [])
+        'datalayer_event': context['request'].session.pop('datalayer_event', [])
```

Comment:
Can you add some context why the default value is a list? It seems like when data is provided we are using an object not in a list.

Reply:
Thanks for the catch @tbrlpld! I updated it to be an empty object instead of a list to avoid confusion 👍